source/auth/tests/mongodb-oidc.rst

```diff
@@ -160,8 +236,67 @@ of listening for SASL commands.
       }
     }
 
-#. Perform another find operation.
-#. Assert that the refresh callback has been called, if possible.
-#. Assert that a ``find`` operation was started twice and a ``saslStart`` operation was started once during the command execution.
-#. Assert that a ``find`` operation succeeeded once and the ``saslStart`` operation succeeded during the command execution.
-#. Assert that a ``find`` operation failed once during the command execution.
+.. note::
+
+  the driver MUST either use a unique ``appName`` or explicitly
+  remove the ``failCommand`` after the test to prevent leakage.
+
+- Perform another find operation that succeeds.
+- Assert that the refresh callback has been called, if possible.
+- Assert that a ``find`` operation was started twice and a ``saslStart`` operation was started once during the command execution.
+- Assert that a ``find`` operation succeeeded once and the ``saslStart`` operation succeeded during the command execution.
```

Comment:
Maybe it's worth clarifying the wording here.  Node's initial implementation counts the number of command started / succeeded events and asserts on event counts, but other scenarios would satisfy these same event counts. (a connection connection + a normal retryable find would produce the same event counts but in the wrong order)

```suggestion
- Assert that the ordering of command started events is `find`, `saslStart` , `find`.
- Assert that the ordering of command succeeded events is `saslStart`, `find`.
```